jenkins/post_install.md

```diff
@@ -0,0 +1,54 @@
+## Jenkins
+
+### External access
+
+By default external access to the Jenkins UI port isn't available. This is easily changed by applying the following YAML to your cluster with `kubectl apply -f jenkins-service.yaml` (or whatever you call the file containing the contents below):
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: jenkins-service
+spec:
+  type: LoadBalancer
+  ports:
+    - port: 8080
+      targetPort: 8080
+      protocol: TCP
+  selector:
+    app: jenkins
+```
+
+This will open up http://YOUR_CLUSTER_ID.k8s.civo.com:8080 to the whole world. You should lock this down in the [firewall](https://www.civo.com/account/firewalls) automatically created in Civo for your Kubernetes cluster. Locking down the firewall will only affect access from OUTSIDE of your Kubernetes cluster, access from your applications within Kubernetes will not be affected.
+
+### Usage instruction
+
+You can use this application from within your cluster by just using the hostname `jenkins`, so jenkins agents can use the hostname `jenkins` and the port `5000`. 
+
+After jenkins has been deployed and you have exposed external access, you can access jenkins on the mentioned url and port, and then jenkins will request the initial password:
+
+<img width="998" alt="5CE566FA-2B7F-42B5-B19F-63D19A4C4BC0" src="https://user-images.githubusercontent.com/567298/66704075-358e6300-ed19-11e9-8973-29c45ce67f20.png">
+
+Get the jenkins pod name:
+
+```
+$ kubectl get pods --selector app=jenkins
+```
+
+Then exec into the container and retrieve the initial password:
+
+```
+$ kubectl exec -it jenkins-xxxxxxxxx-xxxxx -- cat /var/jenkins_home/secrets/initialAdminPassword
+423d1248daee49b8949a73abe52b3a43
+```
```

Comment:
My only initial comment is, is it not possible to specify the initialAdminPassword from within a configmap or something or an ENV variable? Then we can randomly generate one using the `CIVO:ALPHANUMERIC(x)` value and have it presented in the Web UI and CLI? Like https://github.com/civo/kubernetes-marketplace/blob/master/postgresql/manifest.yaml#L29

Reply:
Good point. Let me have a look into it.